cmd/importer/README.md

```diff
@@ -0,0 +1,42 @@
+# Kueue Importer Tool
+
+A tool able to import existing pods into kueue.
+
+## Cluster setup.
+
+The importer should run in a cluster having the Kueue CRDs defined and in which the `kueue-controller-manager` is not running or has the `pod` integration disabled.
```

Comment:
What does mean ```has the `pod` integration disabled```?

Reply:
In short , not doing https://kueue.sigs.k8s.io/docs/tasks/run_plain_pods/#before-you-begin before the import.